deployment/docker_compose/docker-compose.dev.yml

```diff
@@ -74,6 +74,9 @@ services:
       - DISABLE_RERANK_FOR_STREAMING=${DISABLE_RERANK_FOR_STREAMING:-}
       - MODEL_SERVER_HOST=${MODEL_SERVER_HOST:-inference_model_server}
       - MODEL_SERVER_PORT=${MODEL_SERVER_PORT:-}
+      - VESPA_REQUEST_TIMEOUT=${VESPA_REQUEST_TIMEOUT:-}
```

Comment:
If we're going to add these to the deployment files, we should add it to all of them and all of the different ways to deploy. I think in this case if it's a single customer, we can just tell them to add it themselves, should be ok hopefully